src/nova-components/CreateFavorites/resources/js/helpers.js

```diff
@@ -0,0 +1,9 @@
+export function parseUserResponse(response) {
```

Comment:
This method is specifically parsing Ids and names only, we want it to parse every possible field that a response might have, so don't treat this as something specific for users